pxc/jenkins/pxc56-pipeline.groovy

```diff
@@ -86,6 +90,18 @@ pipeline {
                 sh '''
                     MY_BRANCH_BASE_MAJOR=5
                     MY_BRANCH_BASE_MINOR=6
+
+                    if [[ ${USE_PR} == "true" ]]; then
+                        if [ -f /usr/bin/yum ]; then
+                            sudo yum -y install jq
+                        else
+                            sudo apt-get install -y jq
```

Comment:
are you sure that we don't need to update apt db here?

Reply:
I am, it'll always use docker executor, which has yum make cache:https://github.com/Percona-Lab/jenkins-pipelines/blob/master/IaC/pxc.cd/init.groovy.d/cloud.groovy#L113

But, APT/DEB as well update their cache: https://github.com/Percona-Lab/jenkins-pipelines/blob/master/IaC/pxc.cd/init.groovy.d/cloud.groovy#L298